### PR TITLE
Minor changes to the schema to get it to validate with a sample feed.

### PR DESCRIPTION
**Pull request by jdmgoogle**
Some of the elements and attributes needed to have some minOccurs/maxOccurs values tweaked.
Also made the FilingDeadline a date instead of a datetime since that's what was in the previous sample feed.

### CODE REVIEW

File: vip_spec.xsd [hunk 1]
```diff
@@ -259,7 +259,7 @@ reference elements for polling location
       <xs:element name="PostElectionStatus" type="CandidatePostElectionStatus" minOccurs="0" />
       <xs:element name="PreElectionStatus" type="CandidatePreElectionStatus" minOccurs="0" />
       <!-- NOTE: Still trying to get feedback or an example of what this means and how it's used -->
-      <xs:element name="SequenceOrder" type="xs:integer" />
+      <xs:element name="SequenceOrder" type="xs:integer" minOccurs="0" />
```
Comment (jktomer) [inline vip_spec.xsd]: As mentioned in discussion on the PR that introduced ballot style IDs to precinct, I think we can safely scratch this altogether, but that's probably a topic for a separate PR.

File: vip_spec.xsd [hunk 2]
```diff
@@ -421,7 +421,7 @@ reference elements for polling location
       <xs:element name="OtherVoteVariationType" type="xs:string" minOccurs="0" />
       <xs:element name="ElectorateSpecification" type="InternationalizedText" minOccurs="0" />
       <!-- NOTE: Still trying to get feedback or an example of what this means and how it's used -->
-      <xs:element name="SequenceOrder" type="xs:integer" />
+      <xs:element name="SequenceOrder" type="xs:integer" minOccurs="0" />
```
Comment (jktomer) [inline vip_spec.xsd]: ditto above.

**Comment by jktomer** [comment]
LGTM

File: vip_spec.xsd (hunk 1 above)
Comment (jdmgoogle) [inline vip_spec.xsd]: Yeah, basically I'm doing the minimum amount of work just to get the schema to a point where a feasible sample feed can validate.

**Comment by jungshadow** [comment]
LGTM

**Pull request merged**